File: example_app/src/main/java/com/rohitss/uceh_example/MainActivity.java
package com.rohitss.uceh_example;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void onCrashButtonClicked(View view) {
        Toast.makeText(this, "App Crashed", Toast.LENGTH_SHORT).show();
        throw new RuntimeException("Hey! App Crashed.");
    }
}

File: example_app/src/main/java/com/jampez/uceh_example/MainActivity.kt
package com.jampez.uceh_example

import android.os.Bundle
import android.view.View
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

@Suppress("UNUSED_PARAMETER")
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState?: Bundle())
        setContentView(R.layout.activity_main)
    }

    fun onCrashButtonClicked(view: View?) {
        Toast.makeText(this, "App Crashed", Toast.LENGTH_SHORT).show()
        throw RuntimeException("Hey! App Crashed.")
    }
}